src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
-body {
+@import './utils/variables';
+@import './blocks/page';
+@import './blocks/header';
+@import './blocks/nav';
+@import './blocks/products';
+@import './blocks/card';
+@import './blocks/stars';
+
+* {
+  box-sizing: border-box;
+}
+
+h2,
+p {
   margin: 0;
 }
```

Comment:
Consider moving general styles to a separate file to keep the main file clean and clear